docs/guide/running-extern-robot-controllers.md

```diff
@@ -54,6 +54,15 @@ Generic Webots environment variables needed for all the controller languages:
 
 %tab-end
 
+%tab "Windows (development)"
+| Environment Variable                | Typical Value                                                             |
+|-------------------------------------|---------------------------------------------------------------------------|
+| WEBOTS\_HOME                        | `C:\tools\msys64\home\username\webots`                                    |
```

Comment:
> Note that this (and `MSYS_HOME`) is a custom path specific for your system but it is written as it should be valid for all the systems.

Yes, but it says `Typical Value`, e.g. this is also not correct: `WEBOTS_HOME     C:\Program Files\Webots`   

> I'm also not sure that instructions for development installations should go in the User Guide.
> I would rather add it in the GitHub wiki.

I would also hide this from users, but I afraid it will be hard to find it (for us and for users who want to develop)

Reply:
> Note that this (and MSYS_HOME) is a custom path specific for your system but it is written as it should be valid for all the systems.
This could be misleading for users.

Maybe we could remove the `tools` folder?

Reply:
> I would also hide this from users, but I afraid it will be hard to find it (for us and for users who want to develop)

If you add it in the installation instructions page then they are visible for users and every users that want to install the development system will see them.


> Maybe we could remove the `tools` folder?

Yes, when installing MSYS_2 the proposed folder is not `tools` by default.